TASK/TaskCower.md

```diff
@@ -10,6 +10,7 @@ void TASK_COWER(Ped ped, int duration);
 
 
 ## Parameters
-* **ped**: 
-* **duration**: 
+* **ped**: Ped id.
+* **duration**: Duration of the animation in ms. Set to -1 if you do not want a duration.
 
+The ped will act like NPC's involved in a gunfight. The ped will squat down with their heads held in place and look around.
```

Comment:
Please move this line above the Parameters section to make it actually appear in the description of this native. It will be ignored if it's in the current spot.